pub/helpers/bin/dependency_services.dart

```diff
@@ -0,0 +1,80 @@
+/// Support for automated upgrades.
+///
+/// This
```

Comment:
minor: were there some additional docs to be added here?

Reply:
Hmm - no longer remember what I wanted to write.

Removed